oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/OAuth2AuthorizationCodeGrantFilter.java

```diff
@@ -134,6 +134,17 @@ public final void setAuthorizationRequestRepository(AuthorizationRequestReposito
 		this.authorizationRequestRepository = authorizationRequestRepository;
 	}
 
+	/**
+	 * Sets the {@link RequestCache} used for storing the current request
+	 * before redirecting the OAuth 2.0 Authorization Request.
```

Comment:
The javadoc needs to be updated here as it does not save the current request. Instead it loads a saved request and replays it.